src/script/page/VerifyEmailAccount.tsx

```diff
@@ -44,7 +44,9 @@ const VerifyEmailAccount = ({location}: Props) => {
   const {accountAction} = useContext(ActionContext);
   const redirectPhone = (Runtime.isAndroid() || Runtime.isIOS()) && REDIRECT_VERIFY_URL;
   const redirectDesktop =
-    Runtime.isDesktopOS() && !(Runtime.isMacOS() || Runtime.isWindows()) && `${WEBAPP_URL}/auth/?immediate_login#login`;
+    Runtime.isDesktopOS() &&
+    !(Runtime.isMacOS() || Runtime.isWindows() || Runtime.isLinux()) &&
```

Comment:
Maybe this would be easier?

```suggestion
    !Runtime.isMacOS() && !Runtime.isWindows() && !Runtime.isLinux() &&
```

Reply:
Let's assign a named variable to expressions like `!Runtime.isMacOS() && !Runtime.isWindows() && !Runtime.isLinux()`